BackendBench/backends/llm_relay.py

```diff
@@ -35,6 +34,26 @@ def __init__(self, model: str = "gcp-claude-4-sonnet") -> None:
         self.kernels_dir = f"generated_kernels/llm_relay_run_{timestamp}"
         os.makedirs(self.kernels_dir, exist_ok=True)
 
+        # Determine configuration based on USE_PLUGBOARD environment variable
+        use_plugboard = os.getenv("USE_PLUGBOARD", "0") == "1"
```

Comment:
seems kinda involved? My initial hope was that this should be supported by effectively removing some authentication or hitting a different http endpoint. Introducing yet another backend seems off

Reply:
Granted something we could do is just point people to this PR if they want a simple agent, directionally I want us to removing all agent code from the repo except if it's for us to test things out quickly